wp-content/themes/core/blocks/core/columns/index.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Scripts specific to this block
+ */
+
+import './style.pcss';
+import './editor.pcss';
```

Comment:
I wonder if we skip an editor.pcss unless it's needed?

Reply:
I'm good w/ that. It doesn't appear that block editor styles are getting enqueued in the admin at all right now. If there was a way to intelligently not enqueue if the file is empty, I'd say keep them because if they're needed it's less setup. But if we're loading a bunch of empty stylesheets, then yea, lets only do it when necessary.

LendNation did end up w/ a bunch that are empty. Only a small handful (5?) needed specific editor styles.